packages/cells/src/model.ts

```diff
@@ -20,7 +20,7 @@ import {
 } from '@jupyterlab/coreutils';
 
 import {
-  IObservableJSON, IModelDB, IObservableValue, ObservableValue
```

Comment:
This was just renaming the `ObservableValue` namespace to `IObservableValue` for consistency with the other `IObservables`. I can revert that.